src/Avalonia.Base/Animation/CrossFade.cs

```diff
@@ -132,32 +132,23 @@ public async Task Start(Visual? from, Visual? to, CancellationToken cancellation
             }
 
             var tasks = new List<Task>();
-            using (var disposables = new CompositeDisposable(1))
-            {
-                if (to != null)
-                {
-                    disposables.Add(to.SetValue(Visual.OpacityProperty, 0, Data.BindingPriority.Animation)!);
-                }
 
-                if (from != null)
-                {
-                    from.Opacity = 0f;
```

Comment:
Another option would be to replace from.Opacity with the same:
```
disposables.Add(from.SetValue(Visual.OpacityProperty, 0, Data.BindingPriority.Animation)!)
```

Which should be reverted later, when disposables are disposed. But either way, your PR looks good to me.

Reply:
Thanks.

This was my original solution but realised it could be simplified and assumed removing the disposables would be more performant.